kedro/templates/project/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/settings.py

```diff
@@ -3,29 +3,27 @@
 https://kedro.readthedocs.io/en/stable/kedro_project_setup/settings.html."""
 
 # Instantiated project hooks.
-# from {{cookiecutter.python_package}}.hooks import ProjectHooks
-# HOOKS = (ProjectHooks(),)
+# HOOKS = (,)
 
 # Installed plugins for which to disable hook auto-registration.
 # DISABLE_HOOKS_FOR_PLUGINS = ("kedro-viz",)
 
 # Class that manages storing KedroSession data.
-# from kedro.framework.session.shelvestore import ShelveStore
-# SESSION_STORE_CLASS = ShelveStore
+# from kedro_viz.integrations.kedro.sqlite_store import SQLiteStore
+# from pathlib import Path
+# SESSION_STORE_CLASS = SQLiteStore
```

Comment:
The ShelveStore isn't really useful and create a lot of confusion before. SQLiteStore is the most common one, however user need to install kedro-viz first.

Reply:
I think it's a sensible choice